packages/automerge-doc-server/src/server.ts

```diff
@@ -133,14 +148,18 @@ export class AutomergeServer implements SocketIOHandlers {
         //
         // XXX: frontend/src/api/document.ts needs to be kept up to date with this
```

Comment:
Given the comment, I have to ask the obvious question: does the frontend migration code need to be updated in light of this change?

I'm not blocking this PR on that, but if it is a problem, please take appropriate action by filing an issue or making a followup PR.

Reply:
I didn't think it needed to be updated, since we are just wrapping the migrateDocument function in `automerge-doc-server` to make it slightly easier to use in that context. The rest of the process for migrating a doc in TS (getting the migrated doc, creating a diff, applying a diff), remains the same.

However this is now messier than a simple cut and paste. Unfortunately I don't see an easy way of factoring out the common code without importing either `frontend` or `automerge-doc-server` into the other.

I'm inclined to let this slide, but would understand if you felt differently.